tests/apollo/test_skvbc.py

```diff
@@ -95,7 +95,8 @@ class SkvbcTest(unittest.TestCase):
         Ensure that we can put a block and use the GetBlockData API request to
         retrieve its KV pairs.
         """
-        bft_network.start_all_replicas()
+        if bft_network.pre_built_network is False:
+            bft_network.start_all_replicas()
```

Comment:
Instead of this, let's modify the `start_all_replicas()` method itself. If it catches an `AlreadyRunningError`, rethrow it only if `is_existing = False`. This way we won't have to modify every single test that starts all replicas.

Reply:
Don't do `raise e` but `raise` only instead.